crates/re_types/definitions/rerun/testing/components/fuzzy_deps.fbs

```diff
@@ -0,0 +1,21 @@
+include "arrow/attributes.fbs";
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+
+namespace rerun.testing.components;
+
+struct PrimitiveComponent (
+  "attr.rust.derive": "PartialEq",
+  "attr.rust.repr": "transparent",
+  order: 001
+) {
+  value: uint (order: 100);
+}
+
+table StringComponent (
+  "attr.rust.derive": "PartialEq",
+  "attr.rust.repr": "transparent",
+  order: 002
+) {
+  value: string (order: 100);
+}
```

Comment:
A comment reminding everyone that both of these are implicitly arrow-transparent (and that is the point of the test case) would be nice